src/App.tsx

```diff
@@ -25,7 +27,17 @@ function App() {
     if (nuid) {
       setUser(nuid);
     }
-  }, []);
+  }, [])
+
+  const SampleEvent = {
```

Comment:
Nit: Instead of having to cast this object later in your code, you can instantiate it as an Event, like below:
```suggestion
  const SampleEvent: Event = {
```

I recommend this syntax a lot in TypeScript since it really enforces types 😭